reference/docs-conceptual/developer/cmdlet/approved-verbs-for-windows-powershell-commands.md

```diff
@@ -19,6 +19,17 @@ performed.
 > a standard verb in the English language. For example, the term _New_ is a valid PowerShell verb
 > name because it implies an action even though it is not a verb in the English language.
 
+<!--
+01234567890123456789012345678901234567890123456789012345678901234567890123456789
+-->
+
+Each approved verb has a corresponding _alias prefix_ defined.
```

Comment:
I don't know if we need an entire different section. If you noticed the first point in the verb naming rules, it references the alieases that are defined in the table. If you don't think that is clear enough we might wasnt to try and rephrase that section.

Reply:
That comment is out of place there and confuses the reader.  That section is about naming _commands_, not _aliases_.  There is no section about naming aliases and, given how weak the recommendation is, making one would be excessive.  Thus I concluded that a paragraph in the introduction will be most appropriate.